rust/extractor/src/translate.rs

```diff
@@ -928,20 +931,27 @@ impl CrateTranslator<'_> {
             }
             ModuleDef::Function(function) => {
                 let def: ra_ap_hir::DefWithBody = function.into();
-                let (body, source_map) = self.db.body_with_source_map(def.into());
-                let txt = body.pretty_print(self.db, def.into(), Edition::Edition2021);
-                println!("{}", &txt);
 
                 let name = function.name(self.db);
-
                 let location = self.emit_location(function);
-                let body = self.emit_expr(body.body_expr, &body, &source_map);
+
+                let body = if self.extract_dependencies || self.krate.origin(self.db).is_local() {
+                    let (body, source_map) = self.db.body_with_source_map(def.into());
+                    let txt = body.pretty_print(self.db, def.into(), Edition::Edition2021);
+                    println!("{}", &txt);
```

Comment:
this is a debug print I guess?

We might want to sprinkle some `trace!` logs maybe at some point (not in scope now I think)

Reply:
Yes, this should be removed at some point. I found it quite useful to see the expanded HIR code.

Reply:
in a previous iteration of the code that was not using the `trap` library, I was `trace!`ing everything emitted. Might be worth replacing the `println!` with that and keeping it in. There should be a way to get verbose mode out with env variables

Reply:
(but then again, this can be done as a follow-up, we can keep the `println!` now)

Reply:
> There should be a way to get verbose mode out with env variables

yeah, just `CODEQL_EXTRACTOR_RUST_VERBOSE=2`